launch_ros/launch_ros/actions/node.py

```diff
@@ -115,9 +120,16 @@ def __init__(
         passed in in order to the node (where the last definition of a
         parameter takes effect).
 
+        .. warning:: Parameters `node_name` and `node_namespace` are deprecated.
+           Use `name` and `namespace` instead.
+
         :param: node_executable the name of the executable to find if a package
             is provided or otherwise a path to the executable to run.
         :param: package the package in which the node executable can be found
+        :param: name the name of the node
+        :param: namespace the ROS namespace for this Node
+        :param: exec_name the label used to represent the process.
+            Defaults to the basename of node executable.
         :param: node_name the name of the node
         :param: node_namespace the ros namespace for this Node
```

Comment:
I would add a comment here indicating deprecation. So, it's more obvious for someone that jumped to the documentation of these arguments.

Reply:
Done e069970